### PR TITLE
Mazda 3 2014 Fingerprint

### PR DESCRIPTION
**Pull request by Jafaral**

### CODE REVIEW

File: selfdrive/car/mazda/values.py [hunk 1]
```diff
@@ -59,6 +59,11 @@ class Buttons:
   ],
 
   CAR.Mazda3: [
+    # Mazda 3 2014
+    {
+      80: 8, 117: 8, 118: 8, 120: 8, 121: 8, 130: 8, 134: 8, 138: 8, 145: 8, 154: 8, 155: 8, 157: 8, 158: 8, 159: 8, 253: 8, 304: 8, 305: 8, 357: 8, 358: 8, 359: 8, 512: 8, 514: 8, 522: 8, 529: 8, 533: 8, 535: 8, 539: 8, 540: 8, 541: 8, 542: 8, 543: 8, 552: 8, 576: 8, 577: 8, 605: 8, 628: 8, 832: 8, 976: 8, 977: 8, 978: 8, 1034: 8, 1045: 8, 1056: 8, 1061: 8, 1067: 8, 1070: 8, 1078: 8, 1086: 8, 1088: 8, 1093: 8, 1108: 8, 1114: 8, 1115: 8, 1116: 8, 1143: 8, 1147: 8, 1154: 8, 1157: 8, 1160: 8, 1163: 8, 1166: 8, 1170: 8, 1177: 8, 1180: 8, 1232: 8, 1233: 8, 1234: 8, 1235: 8, 1236: 8, 1237: 8, 1238: 8, 1239: 8, 1240: 8, 1241: 8, 1242: 8, 1243: 8, 1244: 8, 1246: 8, 1247: 8, 1264: 8, 1266: 8, 1267: 8, 1271: 8, 1272: 8, 1274: 8, 1275: 8, 1277: 8, 1409: 8, 1416: 8, 1425: 8, 1430: 8, 1440: 8, 1479: 8
+    },
+
     # Mazda 3 2017
     {
```
Comment (adeebshihadeh) [inline selfdrive/car/mazda/values.py:68]: can this new fingerprint be merged with one of the other Mazda 3's? if possible, we should merge all the mazda 3 fingerprints

Comment (Jafaral) [reply selfdrive/car/mazda/values.py:68]: Merge?  as in find one common supper fingerprint ?

Comment (adeebshihadeh) [reply selfdrive/car/mazda/values.py:68]: yes

Comment (Jafaral) [reply selfdrive/car/mazda/values.py]: I merged the new one with the old one. The third belongs to a Mazda6, not  Mazda 3, I could get a drive last time I tried to split Mazda6 on its own.

**Pull request merged**